images/primary_node/Dockerfile

```diff
@@ -26,8 +29,21 @@ RUN . /tmp/${REPO_URL_INSERT_SCRIPT} ${MAPR_VERSION} ${MEP_VERSION}
 
 RUN rpm --import http://package.mapr.com/releases/pub/maprgpg.key
 
-RUN yum -y install java-1.8.0-openjdk-devel \
-    mapr-cldb \
+RUN wget --no-check-certificate \
+        --header "Cookie: oraclelicense=accept-securebackup-cookie" -O /tmp/jdk.tar.gz \
+        "${JDK_URL}" && \
+    mkdir -p /usr/java && \
+    tar xf /tmp/jdk.tar.gz -C /usr/java && \
+    rm /tmp/jdk.tar.gz
```

Comment:
Since StreamSets supports OpenJDK, who install Oracle JDK?

Reply:
Hi Rupal, Good question. When I did code this piece, SDC was not supported on OpenJDK that time. So I had to install Oracle JDK. I have no particular preference at this point as StreamSets supports both Oracle JDK and Open JDK now.

Do you have any preference? That will help me to switch back to OpenJDk (which we previously had) or keep it as is.

Reply:
In my opinion, use Open JDK that comes pre-packaged with MapR.